Show switch icon on showcases with switch ports, hide other platforms when not used

Geokureli opened the pull request:
Congrats to @And-0 for launching Go! Go! PogoGirl on Nintendo Switch!

RichardBray commented:
Annoyingly the code is refusing the build locally on my machine. Instead of adding a new icon, could we try replacing one to see if ti makes a difference. Like the steam with the switch one?

Geokureli commented:
The previous title was misleading. the switch svg was [already there](https://github.com/HaxeFlixel/haxeflixel.com/blob/dev/src/files/images/targets/switch-logo.svg). I just added places that use it. furthermore, it seemed to work [the first time I added it](https://github.com/HaxeFlixel/haxeflixel.com/pull/192/commits/8583724b982cc4fb1c9be710891e7ba2bdae4e05) it wasn't until the last commit that it started failing, where I added switch to the "top" showcases and switched other target icopns to be hidden when unused
